src/vellum/workflows/events/types.py

```diff
@@ -15,11 +15,6 @@
     from vellum.workflows.workflows.base import BaseWorkflow
 
 
-class WorkflowEventType(Enum):
```

Comment:
This filter allows us to remove this enum which we were anticipating for some time